src/VisualStudio/Core/Impl/ProjectSystem/CPS/AdditionalPropertyNames.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.CPS
+{
+    internal static class AdditionalPropertyNames
+    {
+        public const string DefaultNamespace = "rootnamespace";
```

Comment:
This seems like it's deserving a of a very long comment why it's like this. :smile:

Reply:
Good point

Reply:
Just change the const name to `RootNamesapce` for the sake of clarity